templates/LoginForm.php

```diff
@@ -231,20 +231,6 @@ class="float-l erroricon" alt="gtk-dialog-error"/>
                                        value="<?php echo htmlspecialchars(@$this->data['imageU2F'] ?: "", ENT_QUOTES) ?>"/>
                                 <input type="hidden" name="imageWebauthn"
                                        value="<?php echo htmlspecialchars(@$this->data['imageWebauthn'] ?: "", ENT_QUOTES) ?>"/>
-
-                                <?php
-                                // If enrollToken load QR Code
-                                if (isset($this->data['tokenQR']))
-                                {
```

Comment:
where will the qr code from enroll via multichallenge be placed?

Reply:
in one of the image inputs. message also comes from the server and this extra fields isn't needed anymore.